data/mods/blazemod/diamond.json

```diff
@@ -0,0 +1,359 @@
+[
```

Comment:
diamond.json was built from all of the diamond items/parts/recipes from blaze_test.json

Reply:
now also split into respective separations